source/STM32firmware/Atari2600Cart/stm32f4_flash.ld

```diff
@@ -40,7 +40,7 @@ _Min_Stack_Size = 0x400; /* required amount of stack */
 /* Specify the memory areas */
 MEMORY
 {
-  FLASH (rx)      : ORIGIN = 0x08000000, LENGTH = 1024K
+  FLASH (rx)      : ORIGIN = 0x08000000, LENGTH = 64K
```

Comment:
Add comment to explain that Firmware is restricted to 64K because the rest of the FLASH is reserved for cartridge use.